scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +526,124 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    The negative hypergeometric distribution models the probability
+    of :math:`k` successes in a sample of exactly :math:`r` failures
+    taken without replacement where :math:`N` are the total number
```

Comment:
"failures taken w/o replacemebt" sounds strange. I find the example in the tutorial easier to understand

Reply:
Is it fine now? I decided to use the example from tutorial for simplicity...